src/Runtime/OMInstrument.inc

```diff
@@ -159,6 +159,21 @@ void ReportMemory() {
 }
 #endif
 
+FILE *getInstrumentFile() {
+  if (fout)
+    return fout;
+  fout = stdout;
+  if (getenv("ONNX_MLIR_INSTRUMENT_FILE")) {
+    char *fileName = getenv("ONNX_MLIR_INSTRUMENT_FILE");
+    FILE *newFileHandle = fopen(fileName, "w");
```

Comment:
When do you close it?

Reply:
yes, I don't like it. There is an OMInstrumentInit, but it is never called. Ideally we should have an OMInstrumentClose.

Would you be able to look into this? And if you do, I would consider renaming omInstrumentInit, to match all the other calls.

Reply:
Should also add that the normal path has no issue, it's only when we call profiling or instrumentation.

Reply:
It would be help if you can create an issue to remind us about the missing of close. Thanks!